### PR TITLE
Update home.md

### PR DESCRIPTION
**Pull request by regencoffeegirl**
Added updates to the docs

### CODE REVIEW

**Review by ronnyhaase — APPROVED** [review]
Looks good to me :)

File: docs/home.md [hunk 1]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
 
-Liquid staking is a method of staking that allows stakers to unstake without
-the typical 2-3 day withdrawal period.
+#### Step 1: Depositing SOL with Sunrise Stake
+The staking process begins with you depositing your SOL via Sunrise Stake’s app.
 
-Sunrise proxies the [Marinade Finance](https://docs.marinade.finance/) protocol,
-which allows liquid staking through the use of liquidity pools.
+Your SOL will be deposited in Sunrise Stake’s Treasury Account, which is controlled by a smart contract (called Treasury Controller). 
+The Treasury Account is a Programme Derived Account (PDA) that is controlled by the Treasury Controller program; this means that only the Treasury Controller program can move the funds in the Treasury Account.
```
Comment (dankelleher) [inline docs/home.md:94]: Program instead of Programme (the US spelling is exclusively used for software programs)

File: docs/home.md [hunk 2]
```diff
@@ -2,49 +2,76 @@
...
+
+#### Supporting carbon offset projects comes at no cost
+Users deposit their SOL with Sunrise Stake and receive gSOL (green SOL) in return as a wrapped token representing the total amount staked. 
+gSOL is a synthetic derivative of SOL, and can be used in the same way as SOL, by protocols or recipients that support it.
+For example, if you are asked to pay a recipient in SOL, you can pay them in gSOL instead. The recipient can then choose to either keep the gSOL, or unstake it and receive SOL.
+The gSOL to SOL ratio is always 1:1 thus stakers experience no fund loss in the process. Additionally, gSOL is a fully fungible token - stakers can continue to trade it, leaving their SOL to earn rewards for nature-based projects!
+
+#### Transparent unstaking 
+
+The deposited stake retains full liquidity and can be traded back to SOL by  the user any time. At present, no fees (beyond the Solana network fee of 5000 lamports) apply to unstaking high values when the user withdraws with a delay of 2-3 days, otherwise a fee of 0.3% is incurred for immediate withdrawals. This is done to secure sufficient liquidity of the staking pool.
```
Comment (dankelleher) [inline docs/home.md:41]: This is now out of date. Please replace with:

Users may withdraw their SOL instantly with no fee, beyond the Solana network fee of 5000 lamports (< $0.01), up to a limit, after which a fee of 0.03% is incurred. 

Alternatively, users may withdraw with no limit with a delay of 2-3 days. This is done to secure sufficient liquidity of the staking pool.

File: docs/home.md [hunk 3]
```diff
@@ -2,49 +2,76 @@
...
+
+#### Transparent unstaking 
+
+The deposited stake retains full liquidity and can be traded back to SOL by  the user any time. At present, no fees (beyond the Solana network fee of 5000 lamports) apply to unstaking high values when the user withdraws with a delay of 2-3 days, otherwise a fee of 0.3% is incurred for immediate withdrawals. This is done to secure sufficient liquidity of the staking pool. 
+
+#### Liquid Staking
+Liquid staking is a method of staking that allows stakers to unstake without the typical 2-3 day withdrawal period.
+Sunrise proxies the [Marinade Finance](https://docs.marinade.finance/) and [SolBlaze] (https://stake-docs.solblaze.org/) protocols, which allows liquid staking through the use of liquidity pools.
+
+_NOTE: In the future, Sunrise Stake aims at offering entirely feeless liquid unstaking. Sunrise will also proxy additional staking protocols._
```
Comment (dankelleher) [inline docs/home.md:47]: Synthesise this section with the above. The Note is no longer fully applicable, now that we have (at least up to a limit) feeless withdrawals.

File: docs/home.md [hunk 4]
```diff
@@ -2,49 +2,76 @@
...
 
-Regenerative economics is a theoretical framework for understanding and
-guiding economic activity that is focused on creating positive social and ecological outcomes.
+#### The longer the staking, the higher the reward
+Keeping your SOL staked in Sunrise as long as possible maximises the climate-positive impact of your SOL, as well as the security of the Solana blockchain.
+This is because the longer you stake your SOL, the more yield it will accrue, thus increasing the funds directed towards climate-positive causes.
+The decision to unstake is entirely up to the user however the accrued yield is being directed towards offsetting carbon throughout the entire staking period, not only upon unstaking. This way stakers have the possibility to passively invest in green projects while retaining access to their tokens at the same time.
+
+
+### What are carbon tokens?
```
Comment (dankelleher) [inline docs/home.md:58]: Consider mentioning and linking NCT here?

File: docs/home.md [hunk 5]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
+Let’s take a deep dive into what happens when you deposit your SOL with Sunrise Stake
 
-Liquid staking is a method of staking that allows stakers to unstake without
-the typical 2-3 day withdrawal period.
+#### Step 1: Depositing SOL with Sunrise Stake
+The staking process begins with you depositing your SOL via Sunrise Stake’s app.
 
-Sunrise proxies the [Marinade Finance](https://docs.marinade.finance/) protocol,
-which allows liquid staking through the use of liquidity pools.
+Your SOL will be deposited in Sunrise Stake’s Treasury Account, which is controlled by a smart contract (called Treasury Controller).
```
Comment (dankelleher) [inline docs/home.md:93]: This is not quite right - it is not deposited into a treasury account, it is deposited in Marinade and Blaze stake pools . The resultant pool tokens are kept in a PDA controlled by the sunrise stake smart contract. 

The treasury account and treasury controller are not involved at this point. They only get involved once the yield is extracted from the pools.

File: docs/home.md [hunk 6]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
 
-### What is gSOL?
+When you deposit your SOL, you will also receive an equivalent amount of gSOL in your wallet.
+You can either keep your gSOL in your wallet, or use it on platforms that support it.
 
-gSOL is a synthetic derivative of SOL, which is minted by the Sunrise Stake protocol
-when you stake SOL. It is a 1:1 representation of SOL, and can be used in the same
-way as SOL, by protocols or recipients that support it.
+#### Step 2: Sending your SOL to a staking protocol
+Once your SOL is deposited, the Treasury Controller program will send your SOL from the Treasury Account to non-custodial liquid staking protocols such as Marinade Finance (Marinade) and SolBlaze.
```
Comment (dankelleher) [inline docs/home.md:100]: This is done by the sunrise program and not the treasury controller program.

File: docs/home.md [hunk 7]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
 
-gSOL is a synthetic derivative of SOL, which is minted by the Sunrise Stake protocol
-when you stake SOL. It is a 1:1 representation of SOL, and can be used in the same
-way as SOL, by protocols or recipients that support it.
+#### Step 2: Sending your SOL to a staking protocol
+Once your SOL is deposited, the Treasury Controller program will send your SOL from the Treasury Account to non-custodial liquid staking protocols such as Marinade Finance (Marinade) and SolBlaze.                                        
 
-For example, if you are asked to pay a recipient in SOL, you can pay them in gSOL instead.
-The recipient can then choose to either keep the gSOL, or unstake it and receive SOL.
+Treasury Controller will stake your SOL in:
```
Comment (dankelleher) [inline docs/home.md:102]: See above

File: docs/home.md [hunk 8]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
+1. Marinade's stake pool 
+2. Solblaze's stake pool - called BlazeStake 
+3. Marinade's liquidity pool 
 
-Keeping SOL staked in Sunrise as long as possible maximises the climate impact of each stake
-as well as the stability of the Solana blockchain.
+_NOTE_: Sunrise Stake has future plans to integrate other non-custodial staking protocols (in addition to Marinade and SolBlaze).
+ 
+##### Stake Pools 
+When Treasury Controller stakes your SOL in Marinade's stake pool, Marinade mints an amount of msOL tokens (Marinade Staked SOL tokens) representing the size of your holding of the stake pool, which is redeemable for SOL.
```
Comment (dankelleher) [inline docs/home.md:110]: mSOL typo

File: docs/home.md [hunk 9]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
 
-### I am interested in integrating gSOL into my project, how can I do that?
+For example, if Marinade's stake pool contains 20 SOL and the total minted mSOL is 10, then if you add 1 SOl, Marinade mints something like (1/20) * 10 = 0.5 mSOL.
```
Comment (dankelleher) [inline docs/home.md:112]: I think remove this paragraph.

File: docs/home.md [hunk 10]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
+The staking process for SolBlaze is the same as above; when the Treasury Controller program stakes your SOL into BlazeStake's stake pool, Sunrise receives bSOL tokens (BlazeStake Staked SOL tokens) relative to your SOL's size of the stake pool. Just like mSOL, bSOL is redeemable for SOL.
 
-The gSOL token address is `gso12BFMxXD7RJaTUFfZZyuLW32M1hA9LoLZN7u9Und`.
+mSOL and bSOL accrue value 
 
-You can convert the gSOL back to SOL either by unstaking it manually
-in the [Sunrise Stake app](https://app.sunrisestake.com), through the upcoming JS library, or directly
-via the protocol itself.
+##### Liquidity Pool 
+2.  A  mSOL/SOL unstake pool - through which Sunrise becomes a liquidity provider to Marinade
```
Comment (dankelleher) [inline docs/home.md:120]: We should mention here why we do this. We do this to ensure a certain amount of SOL is always liquidly available, (LP unstake is free) while still earning yield on it.

File: docs/home.md [hunk 11]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
+  
+#### Step 3: Accruing yield
+###### Stake Pools
+At the end of each epoch (every 2 or 3 days), yield gets paid out into stake accounts, and the value of the overall stake pool increases. 
+As mSOL and bSOL represent a share of their respective stake pools, the value of mSol and bSOL also increases.
 
-In conjunction with the [Solana Foundation](https://solana.org/), we are exploring
-establishing a green stake pool, composed of validators that use renewable energy for
-their operations.
+The yield acrrued on the staked SOL, is equal to the value of mSOL and bSOL held by Sunrise minus the value of SOL staked (calculated as the circulatinbg supply of gSOL).
```
Comment (dankelleher) [inline docs/home.md:129]: typo accrued

Also remove the comma after SOL

File: docs/home.md [hunk 12]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
-Once the pool is established, Sunrise will direct stake to this pool, resulting in positive
-climate impact on both sides of the transaction:
+###### Liquidity Pool
 
-- Validators have an incentive to use renewable energy, reducing the carbon cost of the Solana network.
-- Stake yield is directed towards offsetting carbon emissions outside the solana network.
 
-### Where does the earned yield go?
+### Unstaking
+If you unstake your SOL, Sunrise calculates how much mSOL to sell back to Marinade to receive your SOL
```
Comment (dankelleher) [inline docs/home.md:135]: "If you unstake your SOL, Sunrise calculates how much of its share of the underlying stake and liquidity pools to sell, in order to receive your SOL"

File: docs/home.md [hunk 13]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
+For example, if you decides to unstake 1 SOL, Sunrise will calculate how much mSOL to sell to Marinade (and bSOL to sell back to BlazeStake), in order to receive back that 1 SOL.
+* Once the latest update is made to send staked SOL to the Marinade Liquidity Pool, Sunrise will be selling back a mix of mSOL and Liquidity Pool tokens to Marinade.
```
Comment (dankelleher) [inline docs/home.md:137]: Remove this line

File: docs/home.md [hunk 14]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
-In the future, yield will be directed to purchasing carbon offset tokens, which will be
-subsequently burned in order to retire the underlying carbon credits.
+Thanks to the yield earned in the staking process, Sunrise can purchase carbon tokens and burn them to offset carbon emissions. 
 
-Long-term, the decision of where to direct yield will be made by the community, governed by
-the [Sunrise Stake DAO](https://app.realms.today/realm/SunriseStake).
+#### 1. Yield transfer to the treasury account
 
-## Governance
+The yield earned from staking SOL is transferred to the __treasury account__. It is a Program Derived Address ([PDA](https://docs.solana.com/developing/programming-model/calling-between-programs#program-derived-addresses)) meaning that it is owned and controlled by a smart contract, not a private key. This smart contract is known as the __treasury controller__. It can trigger a state change to buy or burn carbon tokens, thus retiring underlying carbon credits.
```
Comment (dankelleher) [inline docs/home.md:146]: Some duplication here with the above. IMO the explanation of the treasury management is better suited down here.

File: docs/home.md [hunk 15]
```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
...
 
-:::tip
+#### 2. Purchasing carbon tokens
+The carbon token used in Sunrise Stake is the Nature Carbon Tonne token. NCT represents a tonne of CO2 or equivalent greenhouse gas removed from the atmosphere in various nature-based projects. It is issued on the Polygon and Celo blockchains. More on the NCT can be found on Toucan’s [blog](https://blog.toucan.earth/announcing-nct-nature-carbon-tonne/) or [GitHub](https://github.com/ToucanProtocol/contracts) repository.
 
-Governance for Sunrise Stake is planned for 2023. Stay tuned!
+As the controller cannot purchase tokens directly from a different chain, Sunrise Stake employs a __holding account__ to buy NCT by bridging it to Solana through [Wormhole](https://www.portalbridge.com/#/transfer).
```
Comment (dankelleher) [inline docs/home.md:151]: No need for this line _and_ 158. I think 158 is sufficient. Maybe add the wormhole detail there.

**Comment by dankelleher** [comment]
A general note - I think we need a lot of diagrams - about 50% diagrams to 50% text. So once the diagrams are in, we might want to prune the text a little.

File: docs/home.md (hunk 1 above)
Comment (dankelleher) [reply docs/home.md:94]: Done

File: docs/home.md (hunk 2 above)
Comment (dankelleher) [reply docs/home.md:41]: Done

File: docs/home.md (hunk 3 above)
Comment (dankelleher) [reply docs/home.md:47]: Done

File: docs/home.md (hunk 4 above)
Comment (dankelleher) [reply docs/home.md:58]: Done

**Pull request merged**